engines/scumm/metaengine.cpp

```diff
@@ -249,6 +249,10 @@ bool ScummEngine::hasFeature(EngineFeature f) const {
 		(f == kSupportsQuitDialogOverride && (_useOriginalGUI || !ChainedGamesMan.empty()));
 }
 
+bool Scumm::ScummEngine::enhancementClassActive(int32 cls) {
```

Comment:
I'd name it `enhancementOn()` or `enhancementEnabled()` for brevity, though it is a matter of taste.

Reply:
Thanks, done!